### DISCUSSION TITLE
strange Bluetooth issue

### DISCUSSION THREAD

**Discussion by 0evroger** [nefarius/DsHidMini#223] [Support]
- [ x] I have searched open and closed issues for duplicates
- [ x] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://vigem.org/Community-Support/
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
I've installed the Bluetooth and Dshid drivers as described in the guide also I tested it in Bluetooth connection and I get this error

### Steps to reproduce
- install all the packages
- check if appears the copy-cat controller error
- shows a different warning

**does not connect by Bluetooth

### Screenshots
error
![image](https://user-images.githubusercontent.com/34942206/172279939-e170d9a3-2d5e-4afd-bcb7-24bfe81936b1.png)
that shows in DShidMini control
![image](https://user-images.githubusercontent.com/34942206/172280244-69fedd87-1f51-48d6-92b0-4bad0189951c.png)

### Machine info

**CPU-Architecture:**x64
**Windows version:** Windows 10 21H2
**Software/driver version(s):**  2.2.282

**Comment by nefarius** [comment]
This message has nothing to do with BthPS3, to properly diagnose [check out the FAQ](https://vigem.org/projects/BthPS3/Frequently-Asked-Questions/), and [see what the driver is actually doing under the hood](https://vigem.org/projects/BthPS3/Debugging-the-drivers/).

**Reply by 0evroger** [reply]
ok, I've followed all the [see what the driver is actually doing under the hood](https://vigem.org/projects/BthPS3/Debugging-the-drivers/). and retrieved this:https://drive.google.com/file/d/19SKEIV-Q_zPK1yGh87_s1GtyphHauQ8a/view?usp=sharing

**Reply by nefarius** [reply]
[This is your issue](https://github.com/ViGEm/BthPS3/issues/34). Either edit the registry as a workaround or wait for the next release.

**Reply by 0evroger** [reply]
thanks!
